### PR TITLE
PR: enable custom MaxIterations count for verifyDomainCreated

### PR DESCRIPTION
**Pull request by maggiehe00**
This PR is to improve stability of JRF domain on external Jenkins  by enabling custom MaxIterations count for verifyDomainCreated from 40 to 80.
Latest clean Jenkins run after code refactoring:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-quicktest/1236/

### CODE REVIEW

**Comment by markxnelson** [comment]
I agree with @anpanigr

**Comment by maggiehe00** [comment]
Specifically on this issue we have already had printed out log infor like:
"oracle.kubernetes.operator.utils.TestUtils checkCmdInLoop Pod itjrfpvwlst-domain-58-infraserver1  output does not contain 'Running'  Iteration [38/40], sleeping 10 seconds more
03-23-2020 06:22:03 INFO  "

This log itself gave me the hint the default MaxIterationsPod before this PR was changed to 40 (It had been 100 for a while), which is not enough for JRF domain.

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java [hunk 1]
```diff
@@ -145,6 +145,30 @@ public void verifyDomainCreated() throws Exception {
...
     }
   }
+  
+  /**
+   * Verifies the required pods are created, services are created and the servers are ready.
+   *
+   * @param maxIterations max iteration count
+   * @throws RuntimeException if pods/services of the domain are not created or WLS is not running
+   */
+  public void verifyDomainCreated(int maxIterations) throws Exception {
```
Comment (markxnelson) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java:155]: don't duplicate logic like this.
make one method call the other.

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java:155]: Refactored the code to remove the duplicate logic. This is the common area I'm testing comprehensively. Thanks for the comments.

Comment (maggiehe00) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java:141]: Done

**Pull request merged**